projects/esupervision-and-delius/src/main/kotlin/uk/gov/justice/digital/hmpps/model/PersonalDetails.kt

```diff
@@ -0,0 +1,7 @@
+package uk.gov.justice.digital.hmpps.model
+
+data class PersonalDetails(
+    val crn: String,
+    val name: Name,
+    val dateOfBirth: String
```

Comment:
Is there a reason to not use a LocalDate for the dateOfBirth rather than a string?

```suggestion
    val dateOfBirth: LocalDate
```